components/aws/sagemaker/tests/integration_tests/resources/config/ack-data-quality-job-definition/config.yaml

```diff
@@ -0,0 +1,36 @@
+PipelineDefinition: resources/definition/data_quality_job_defi_v2_pipeline.py
+TestName: ack-data-quality-job-definition
+Timeout: 1000
+Plural: dataqualityjobdefinitions
+JobInputName: data_quality_job_input
+ModelImage: ((XGBOOST_REGISTRY)).dkr.ecr.((REGION)).amazonaws.com/sagemaker-xgboost:0.90-1-cpu-py3
+ModelUrl: s3://((DATA_BUCKET))/model-monitor/xgboost-churn-model.tar.gz
+EndpointCapture: s3://((DATA_BUCKET))/model-monitor/endpoint_capture
+Arguments:
+  region: ((REGION))
+  data_quality_app_specification:
+    imageURI: ((MODEL_MONITOR_IMAGE)).dkr.ecr.((REGION)).amazonaws.com/sagemaker-model-monitor-analyzer
+  data_quality_baseline_config:
+    constraintsResource:
+      s3URI: s3://((DATA_BUCKET))/model-monitor/baselining/data_quality/constraints.json
+    statisticsResource:
+      s3URI: s3://((DATA_BUCKET))/model-monitor/baselining/data_quality/statistics.json
+  data_quality_job_input:
+    endpointInput:
+      localPath: "/opt/ml/processing/input/endpoint"
+      s3InputMode: File
+      s3DataDistributionType: FullyReplicated
+  data_quality_job_output_config:
+    monitoringOutputs:
+    - s3Output:
+        localPath: /opt/ml/processing/output
+        s3URI: s3://((DATA_BUCKET))/model-monitor/reports/data-quality-job-definition/
+      s3UploadMode: Continuous
+  job_resources:
+    clusterConfig:
+      instanceCount: 1
+      instanceType: ml.m5.large
+      volumeSizeInGB: 30
+  role_arn: ((SAGEMAKER_ROLE_ARN))
+  stopping_condition:
+    maxRuntimeInSeconds: 3600
```

Comment:
max runtime is more than test timeout(line 3) in all tests. test timeout should be more than max runtime